### ISSUE TITLE
SHA Restriction not working

### ISSUE THREAD

**Issue by argrau** [lejard-h/google_maps_webservice#85]
When I restrict the API key from Google Cloud to an Android app with my current SHA key, request are not working.

**Comment by zeshuaro** [comment]
My PR #89 has been approved and merged where you can specify your own headers when making the Google API calls. 

Once it's updated on pub.dev (or you can use the master branch on GitHub straightaway), you can use it with the plugin [`google_api_headers`](https://pub.dev/packages/google_api_headers) to get the required headers and you should be able to use app restricted API keys. See below for an example:

```dart
import "package:google_maps_webservice/geocoding.dart";
import 'package:google_api_headers/google_api_headers.dart';

final headers = await GoogleApiHeaders().getHeaders();
final geocoding = new GoogleMapsGeocoding(apiKey: "<API_KEY>", apiHeaders: headers);
```

Hope this solves your issue and Happy New Year!

**Comment by lejard-h** [comment]
Available in last version :) (0.0.19)
